dev-system/dev/user.clj

```diff
@@ -94,8 +98,12 @@
   ;; Uncomment this to force CMR to use Legacy Services
   ; (configure-for-legacy-services)
 
-  (let [s (system/create-system)
-        s (configure-systems-logging s @settings/logging-level)]
+  (let [s (-> (system/create-system)
+              (configure-systems-logging @settings/logging-level)
+              (assoc-in [:apps :search :public-conf] {:protocol "http"
+                                                      :host "localhost"
+                                                      :port 3003
```

Comment:
You should not hardcode the port here, better to look it up from the config. I think it is in the transmit-lib.